packages/react/src/utils/objectUtils.test.ts

```diff
@@ -0,0 +1,28 @@
+import { objectValuesEqual } from './objectUtils';
+
+describe('objectUtils', () => {
+  describe('objectValuesEqual', () => {
+    it('Returns true if objects are equal', () => {
+      const object1 = { a: 1, b: 2 };
+      const object2 = { a: 1, b: 2 };
+      expect(objectValuesEqual(object1, object2)).toBe(true);
+    });
+
+    it('Returns false if objects are not equal', () => {
+      const object1 = { a: 1, b: 2 };
+      const object2 = { a: 1, b: 3 };
+      expect(objectValuesEqual(object1, object2)).toBe(false);
+    });
+
+    it('Compares the first levels only', () => {
```

Comment:
This description was abit vague? Maybe something more along the lines of testing strict equality for objects?